packages/solc-transpiler/src/compiler.ts

```diff
@@ -79,50 +79,52 @@ export const compile = (configJsonString: string, callbacks?: any): string => {
   for (const [filename, fileJson] of Object.entries(res.contracts)) {
     log.debug(`Transpiling file: ${filename}`)
     for (const [contractName, contractJson] of Object.entries(fileJson)) {
-      // Library links in bytecode strings have invalid hex: they are of the form __$asdfasdf$__.
-      // Because __$ is not a valid hex string, we replace with a valid hex string during transpilation,
-      // storing the links re-substituting the __$* strings afterwards
-      const originalRefStrings = getOriginalLinkRefStringsAndSubstituteValidHex(
-        contractJson
-      )
-      log.debug(
-        `Transpiling contract: ${contractName} with valid hex strings for link placeholders.`
-      )
-      const output = transpileContract(
-        transpiler,
-        contractJson,
-        filename,
-        contractName
-      )
-      log.debug(`Transpiled contract ${contractName}.`)
+      if (!!contractJson && !!contractJson.evm && !!contractJson.evm.bytecode && !!contractJson.evm.deployedBytecode) {
```

Comment:
Probably cleaner to replace with an `if(...) continue` instead of a big if statement wrapper.